### ISSUE TITLE
ng-scroll: Does it only support block elements?

### ISSUE THREAD

**Issue by david-hollifield** [angular-ui/ui-utils#232]
I have a list of images that I'm showing in a 
```
<div ng-scroll-viewport>
<ul>
    <li ng-scroll="image in datasource">
        <img src='blah'>
    </li>
</ul> 
</div>
```
where each element is styled as inline-block. Does the ng-scroll directive support this scenario since multiple elements could make up a row and the top/bottom padding would need adjusted accordingly? If not, anyone have plans on adding that functionality?

Thanks!

**Comment by mfeingold** [comment]
@dhilt : can you give me an update on this one?

**Comment by dhilt** [comment]
It was a great story right after this issue was published -- https://github.com/angular-ui/ui-utils/pull/234. 
And it was resolved with this PR: https://github.com/Hill30/NGScroller/pull/43.
So this one can be surely closed too.

**Comment by PowerKiKi** [comment]
Hey @mfeingold, @dhilt. What is the relationship between ui-scroll in this repository and https://github.com/Hill30/NGScroller ? are they some kind of mirrors ? or is one of them outdated ?

Should issues from here moved over there ?

**Comment by mfeingold** [comment]
We do development in our own repository. One of the reasons is that we prefer to do it in coffescript. As soon as we are happy with the stability we transpile the code to JS and publish it here. I just merged  the scroller v 1.1.0 here from the https://github.com/angular-ui/ui-utils/pull/372.

The version in NGScroller repo is a little more bleeding edge for the price of a little less stability

**Comment by PowerKiKi** [comment]
Actual code base of ui-scroll is in https://github.com/Hill30/NGScroller. Please re-submit your issue over there if needed.
